src/main/kotlin/app/revanced/patches/hexeditor/ad/patch/HexEditorAdsPatch.kt

```diff
@@ -0,0 +1,43 @@
+package app.revanced.patches.hexeditor.ad.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.removeInstruction
+import app.revanced.patcher.extensions.replaceInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.hexeditor.ad.annotations.HexEditorAdsCompatibility
+import app.revanced.patches.hexeditor.ad.fingerprints.PrimaryAdsFingerprint
+
+@Patch
+@Name("disable-ads")
+@Description("Disables ads in HexEditor.")
+@HexEditorAdsCompatibility
+@Version("0.0.1")
+
+class HexEditorAdsPatch : BytecodePatch(
+    listOf(
+        PrimaryAdsFingerprint
+    )
+) {
+    override fun execute(context: BytecodeContext): PatchResult {
+        val method = PrimaryAdsFingerprint.result!!.mutableMethod
+        
+        method.replaceInstructions(
+            0,
+            """
+                const/4 v0, 0x1
+                return v0
+            """
+        )
+
+        for(i in 0..4)
```

Comment:
Why this for loop if it isn't used anywhere?

Reply:
To remove useless instructions, as we are already returning from function

Reply:
You have hardcoded 2, the for loop is never used :).

Reply:
Because there are 5 instructions after returning from function, as we are returning on index 1, so we'll remove index 2 five times

Reply:
Right, got it. It all LGTM then